Client/Application/WebServer.swift

```diff
@@ -18,7 +18,7 @@ class WebServer {
     }
 
     func start() -> Bool {
-        return server.running || server.startWithPort(0, bonjourName: nil)
+        return server.running || server.startWithOptions([GCDWebServerOption_Port: 0, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true], error: nil)
```

Comment:
No we still need it. Otherwise when you move the app to thr background and back to foreground, with the app not being hard-suspended, the server port changes *and* the WKWebViews stay alive, which means that the webviews have internal state that points to URLs that may not exist anymore. This option prevents that.